validator_derive/src/quoting.rs

```diff
@@ -495,7 +495,8 @@ pub fn quote_field_validation(
 }
 
 pub fn quote_schema_validation(v: &SchemaValidation) -> proc_macro2::TokenStream {
-    let fn_ident = syn::Ident::new(&v.function, Span::call_site());
+    let fn_ident: syn::Path = syn::parse_str(&v.function).unwrap();
+    //let fn_ident = syn::Ident::new(&v.function, Span::call_site());
```

Comment:
can you add a test and remove the commented out line?

Reply:
oops, I've removed the line
also added tests